lib/promoted/ruby/client/request_builder.rb

```diff
@@ -105,7 +105,12 @@ def log_request_params(include_insertions: true, include_request: true)
             user_info: user_info,
             timing: timing,
             client_info: merge_client_info_defaults,
-            device: @device
+            device: @device,
+            delivery_log: {
```

Comment:
delivery_log is also a repeated DeliveryLog.  This should be an array.

Reply:
done